README.md

```diff
@@ -1,27 +1,94 @@
 # Quickstarts
 
+## Introduction
+
+The repository contains a set of quickstart projects which provide working example
+for specific Narayana transaction manager capabilities.
+The quickstarts can be used as a reference for your own project.
+
+The list of all available quickstart can be found down at this page.
+Each quickstart is categorized with tags that depict areas the quickstart is connected with.
+
+* [List of quickstarts](#list-of-quickstarts)
+* [Quickstarts by tag](#quickstarts-by-tag)
+* [Contacting us](#contacting-us)
+* [Quickstarts in CI environment](#quickstarts-in-ci-environment)
+* [Running a single quickstart](#running-a-single-quickstart)
+* [Running all quickstarts in a single command](#running-all-quickstarts-in-a-single-command)
+
+***TODO: link here quickstarts from WildFly!!!***
+
+## List of quickstarts
+
+Project name | Description | Maven coordinates | Tags
+------------ | ----------- | ----------------- | ----
+[ArjunaCore/txoj](ArjunaCore/txoj/) | Showing how to use the Transactional Object for Java (_TXOJ_) which is the core implementation concept of the Narayana state machine | _o.j.n.arjunacore.quickstart :txoj_ | _[arjuna.core](#arjuna-core-tag), [standalone](#standalone-tag)_
```

Comment:
It's not 100% clear to me how the tagging is working - so if it is just adding links that should be fine so long as we document it as part of the process when contributing a new QS how to link it in

Reply:
+1, I will do so

now I just need to fix the failing tests on windows (it's because I've enabled the `bat` scripts for CI which were not run for long time and there are some issues with it)

Reply:
https://github.com/jbosstm/quickstart/pull/234